mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -178,6 +186,7 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 							Type:         schema.TypeString,
 							Optional:     true,
 							Computed:     true,
+							Default:      "ENTERPRISE",
```

Comment:
I think this must be removed. Was this intentional?

Reply:
Done.